src/application/services/useNote.ts

```diff
@@ -265,6 +273,23 @@ export default function (options: UseNoteComposableOptions): UseNoteComposableSt
     parentNote.value = undefined;
   }
 
+  /**
+   * Format the received note parents into presentation format
+   */
+  function formatNoteParents(): string {
+    if (currentId.value === null) {
+      throw new Error('note id is not defined');
+    }
+    let presentationFormat = '';
+
+    for (let value of noteParents.value) {
+      presentationFormat += getTitle(value.content) + ' > ';
+    }
+    presentationFormat += noteTitle.value;
+
+    return presentationFormat;
```

Comment:
`parentNote` title should be clickable so just merge titles into one string is not a solution. 

Also can't see the case with `RootNoteTitle > ... > CurrentNote`

Reply:
I think we can use anchor tag `<a>` can solve the merge of titles into one string.

Reply:
it's better to do it in a component